src/pacasam/extractors/bd_ortho_vintage.py

```diff
@@ -44,64 +47,69 @@ class BDOrthoVintageExtractor(Extractor):
     """
 
     patch_suffix: str = ".tiff"
```

Comment:
rgb_column, irc_column and pixel_per_meter have been turned into "uppercase constant", maybe patch_suffix should be too, or is there a reason ?